nncf/torch/binarization/binarize_functions.py

```diff
@@ -60,7 +61,8 @@ class DOREFABinarizeFn(torch.autograd.Function):
     @staticmethod
     def symbolic(g, x):
         zero = g.constant(0, [1], 'float')
-        zero = g.op("Unsqueeze", zero, axes_i=[1, 2, 3])
+        axes = g.op("Constant", value_t=torch.tensor([1, 2, 3], dtype=torch.long))
+        zero = g.op("Unsqueeze", zero, axes)
```

Comment:
I think I used a custom rewrite of the `torch.onnx.symbolic_helper._unsqueeze_helper` function for this already some time ago, but I cannot find it in the current code base. Consider reusing that helper as well, if it helps.

Reply:
thanks, reused the helper